distribution/lib/Standard/Saas/0.0.0-dev/src/Strava.enso

```diff
@@ -0,0 +1,33 @@
+from Standard.Base import all
+from Standard.Base.Enso_Cloud.Enso_Secret import as_credential_reference
+
+polyglot java import org.enso.base.enso_cloud.ExternalLibraryCredentialHelper.CredentialReference
+polyglot java import org.enso.saas.strava.StravaService
+
+type Strava
+    ## PRIVATE
+    private Service (strava_service:StravaService)
+
+    ## ICON cloud
+       Initializes the Strava instance using the given credentials file.
+
+       Arguments:
+       - credentials: a Cloud secret or a file containing Strava credentials.
+    initialize : File|Enso_Secret -> Strava
+    initialize credentials =
+        credentials_reference = as_credential_reference credentials
+        Strava.Service (StravaService.new credentials_reference)
+
+    get_user self =
+        strava_get self "https://www.strava.com/api/v3/athlete"
+
+    get_zones self =
+        strava_get self "https://www.strava.com/api/v3/athlete/zones"
+
+    get_stats self =
+        id = self.get_user.decode_as_json.get "id"
+        strava_get self "https://www.strava.com/api/v3/athletes/"+id.to_text+"/stats"
```

Comment:
Let's not prefix with `get_` as we don't normally.

Likewise, please add doc comments.